Assets/_BForBoss/Scripts/Managers/WorldManager.cs

```diff
@@ -19,6 +19,11 @@ public class WorldManager : MonoBehaviour
 
         [Title("Effects")] 
         [SerializeField] private Volume _deathVolume = null;
+        
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+        [Title("Debug")]
+        [SerializeField] private GameObject _debugCanvas;
```

Comment:
wondering why this has to be in WorldManager, why can't this be drag and drop into a scene?

Reply:
My thought Process is that we should'nt need to remember to delete the DebugWindow Prefab before we make a production Build. Like, I want it to "automate" so that we have to remember to delete/add things to the scene before we make a build

Reply:
i agree with the reminder part. However, WorldManager can change quite often. 
Even possible that we don't use this worldmanager for certain "scenes/worlds". 

Shouldn't we leave this in, even when building the prod build? since it won't be compiled by prod anyway? 
And do you think it's better to have the debug stuff handle all this by itself instead of linking it to worldmanager?

Reply:
hmm good point, we can discuss this during the meeting today